src/java/picard/vcf/GenotypeConcordanceCounts.java

```diff
@@ -74,6 +86,76 @@ public void validateCountsAgainstScheme(final GenotypeConcordanceScheme scheme)
     }
 
     /**
+     * Genotype Concordance is the number of times the truth and call states match exactly / all truth and call combinations made
+     * If the GA4GH scheme is being used, any MISSING sites in truth OR call will not be included in the discordance calculations.
+     */
+    public double calculateGenotypeConcordance(final GenotypeConcordanceScheme scheme, final boolean missingSitesFlag) {
+        double numerator = 0.0;
+        double denominator = 0.0;
+
+        scheme.validateScheme();
+
+        final TruthState[] allTruthStates = TruthState.values();
+        final CallState[] allCallStates = CallState.values();
+
+        for (final TruthState truthState : allTruthStates) {
+            for (final CallState callState : allCallStates) {
+                if(!missingSitesFlag && isMissing(truthState, callState)) {
+                    continue;
+                }
+                else {
+                    final TruthAndCallStates truthAndCallStates = new TruthAndCallStates(truthState, callState);
+                    final int count = getCount(truthAndCallStates);
+                    if (truthState.ordinal()==callState.ordinal()) {
```

Comment:
@nh13 @eitanbanks is it better to compare this way, instead of a string comparison? As long as the CallState and TruthState enum's have the entries that can be paired in the same order then their ordinals will match. The order of the entries should never be changed so I think it will work.